### ISSUE TITLE
Invalid option error

### ISSUE THREAD

**Issue by KestrelKitezh** [Lailloken/Lailloken-UI#311]
Hey

Not really sure if I'm the one doing something wrong here, but I've tried a few things to see if I can fix it, with no luck.

![image](https://github.com/Lailloken/Lailloken-UI/assets/44786999/4a7245d0-3920-4227-949d-b070a2909e28)

This happens alot, and I'm not really sure why. He got read/write access, running as admin, and it seem to be a x/y issue. Could it be a multi monitor issue? I'm running a 5 screen setup with 3 horizontal screens, and 2 above. Could this maybe screw with it?

Running beta .8, and newest autohotkey version .37.1

![image](https://github.com/Lailloken/Lailloken-UI/assets/44786999/011b133c-0471-4d01-a780-1ffeca94b445)

**Comment by Lailloken** [comment]
It seems to be having problems with getting the correct screen coordinates to display help-tooltips when hovering over ui-elements in the settings menu.
Could you right-click the script's tray icon and open `Window Spy`? Make sure `Follow Mouse` is un-checked, then open the script's settings menu again and click into it so it's focused.
I need to know if the script is misreading window coordinates and, if so, what the coordinates are.
![image](https://github.com/Lailloken/Lailloken-UI/assets/61888437/e18f1256-2136-48cb-ad62-6f96aacae4aa)

**Comment by KestrelKitezh** [comment]
First attempt, general tab.
![image](https://github.com/Lailloken/Lailloken-UI/assets/44786999/60aaed82-586a-46e6-bdc9-4cad4e7cbb93)


Clicking the Cheat-sheets
![image](https://github.com/Lailloken/Lailloken-UI/assets/44786999/e3d7b486-249c-4863-ac01-46bbb411a4c8)

**Comment by KestrelKitezh** [comment]
Seem to trigger when I hoover over the enable checkbox

**Comment by KestrelKitezh** [comment]
Wonder if this is what your looking for. This is the same spot; with each in focus.
![image](https://github.com/Lailloken/Lailloken-UI/assets/44786999/010cac6b-16bb-4001-bc90-355530d2b3e6)

Mouse same place with script focused (it auto focuses when error trigger).
![image](https://github.com/Lailloken/Lailloken-UI/assets/44786999/9cdc9b85-8a63-4894-ba0e-12176948e64b)

or are the coords relative to anchor?

**Comment by KestrelKitezh** [comment]
Nevermind the coords relative question. I see it now. When it focuses the window changes, so it makes sense. Screen is always the same

**Comment by Lailloken** [comment]
Thanks for the screenshots. I can see a help-tooltip on the first screenshot of the general tab. Does the error-message appear only when switching tabs and hovering stuff, or does it also appear when hovering over buttons/checkboxes in the general tab?

**Comment by KestrelKitezh** [comment]
General tab and betrayal-info seem to work fine. All hoovers work.
Cheat-sheets
Clone-frames
Hotkeys (only the bottom checkbox trigger error (misc keys))
Item-info
Mapping tracker
Screen-checks
search-strings
updater

^ All above failed on all hoovers. I assume its when he tries to generate a tooltip. I find it abit weird though, since the updater tab, I screenshoted that yesterday when I first posted, and it worked. Not now for some reason. I even reloaded the script between each tab test

**Comment by Lailloken** [comment]
Can you send me a screenshot of your display arrangement?
![image](https://github.com/Lailloken/Lailloken-UI/assets/61888437/c2e0f406-802e-44f3-a4c2-8a89684311f2)

I'll try and replicate it on my end. I can't see anything out of the ordinary in the Window Spy coordinates.

**Comment by KestrelKitezh** [comment]
![image](https://github.com/Lailloken/Lailloken-UI/assets/44786999/28a4d313-0cb3-4c88-9508-57cca4f3e0c8)

**Comment by KestrelKitezh** [comment]
technically could disable all other displays just to see if that changes anything, just to save you some trouble. Gimme a minute

**Comment by KestrelKitezh** [comment]
Still the same with only one display. Ironically betrayal info checkbox hoover failed this time xD

**Comment by Lailloken** [comment]
Just tested something: If you check `Follow Mouse` in the Window Spy and repeatedly click any tab in the settings menu, the `ahk_id` in the top-most box should change its value with every click but everything else should stay the same (some values in the `mouse position` box may flash briefly, but that doesn't matter). Could you check if it behaves the same on your end?

***
Something else entirely:

![image](https://github.com/Lailloken/Lailloken-UI/assets/61888437/2dbf7b10-e5d0-4e5a-b1a8-007580e46e02)

Just noticed how your Window Spy looks different between screenshots. Scaling seems to be bricked or something. May not be significant, but I'm grasping at straws here. The fact that the error pops up inconsistently between tabs could also point to some AutoHotkey bug. You could try a clean reinstall.

**Comment by KestrelKitezh** [comment]
Thats because I tried to drag it out I think to see the entire error message in the all text box. The spy thingie broke alittlebit when I started resizing it

**Comment by KestrelKitezh** [comment]
can uninstall and try anyway though just to check

**Comment by Lailloken** [comment]
I couldn't get this weird issue out of my head, so here's something you can try. It's a modified version of the script that shows the screen-coordinates. If you hold the shift-key and hover over checkboxes/buttons, a text-box will open:
![image](https://github.com/Lailloken/Lailloken-UI/assets/61888437/74bec4f6-8702-4bd3-acd8-c215458268e8)

[Lailloken UI.zip](https://github.com/Lailloken/Lailloken-UI/files/12567853/Lailloken.UI.zip)

**Comment by KestrelKitezh** [comment]
Enabled cheatsheet, and wh en I hoovered over hotkey settings :
![image](https://github.com/Lailloken/Lailloken-UI/assets/44786999/283900c2-1889-4577-876e-3f54b169101c)

**Comment by Lailloken** [comment]
Are you using any software/hardware solution for your monitor setup, or do they all just go straight into your graphics card? The script cannot reliably get the position of ui-elements that are being hovered over.

**Comment by KestrelKitezh** [comment]
straight into gpu. Nothing other than windows handle it

**Comment by KestrelKitezh** [comment]
But this seem so far to be a only me problem though (?), so maybe it isn't worth spending time on.

**Comment by Lailloken** [comment]
I'll release an update tomorrow morning that uses the cursor-position for tooltips as a fallback if the normal coordinates cannot be retrieved correctly.

**Comment by Lailloken** [comment]
Since these tooltips are also used outside the settings menu, more testing needs to be done. Use this modified version for now:
[Lailloken UI.zip](https://github.com/Lailloken/Lailloken-UI/files/12572704/Lailloken.UI.zip)

The easiest way to test them outside the settings menu is enabling the `Lab Overlay` in the `Minor QoL Tools` tab, then entering Aspirants' Plaza and holding TAB. An overlay with an icon in the top-left corner will open, and that icon has a tooltip when hovered over.

**Comment by KestrelKitezh** [comment]
Hoovers in the lab overlay works fine

**Comment by KestrelKitezh** [comment]
Please note that this new file broke the F1 key in clone-frame's

**Comment by KestrelKitezh** [comment]
![image](https://github.com/Lailloken/Lailloken-UI/assets/44786999/1968029d-b589-4b56-855a-679cd949c206)

Tried to save. That might be my fault though, since I fiddled pretty hard with the frame

**Comment by Lailloken** [comment]
Something seems fundamentally broken. The modified file doesn't handle anything with hotkeys, so it's strange that the F-keys are bricked. But generally this looks like there are problems with screen/mouse coordinates yet again, with F3 only moving on one axis.

The error-message after saving is due to the script not being able to refresh/rebuild the settings window correctly. It normally creates a second window on top of the first one, then destroys the first one in order to prevent the window from causing a flash. In your case, it can't find the first one anymore and throws an error.

Did you end up reinstalling AutoHotkey two days ago, when I suggested it?

**Comment by KestrelKitezh** [comment]
I did uninstall, then rebooting and installed again. And nothing is modified in windows except murdering the telemetry services windows is shipped with these days.

On a sidenote, I did notice something different now though. I reloaded the script, and suddenly F1 works, but F3 doesn't work vertically now. Only horizontally. There is something very weird going on here. Positive note is that I cant manage to get an error no matter how hard I try.
